portfolio/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-  <application>YOUR_PROJECT_ID_HERE</application>
+  <application>ashoupherrera-sps-spring20
+</application>
```

Comment:
This is just a small comment (usually referred as "nit"), and it may be just OK to keep this as is, but just mentioning mostly for minor improvements/beautifications in the future. No need to put the closing tag (`</application>`) in the new line. In fact, I'm not sure what the exact consequence of this (putting the closing tag in the new line). I suppose your AppEngine application deployment (`mvn appengine:update`) might fail, but it may be OK. Let's give this try as is and fix if it doesn't work as expected. I'll approve this PR and merge for the sake of moving forward. Thanks for making a quick turnaround!